packages/nocodb/src/helpers/dataHelpers.ts

```
@@ -237,6 +237,11 @@ export async function serializeCellValue(
         return Number(value).toFixed(column.meta?.precision ?? 1);
       }
       break;
+    case UITypes.Duration:
+      {
+        return convertMS2Duration(value, column.meta.duration);
+      }
+      break;
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)